pkg/terraform/terraform_mock.go

```diff
@@ -83,6 +84,15 @@ func (m *MockTerraformClient) Show(ctx context.Context, stdout, stderr io.Writer
 	return 0, nil
 }
 
+func (m *MockTerraformClient) Format(ctx context.Context, stdout, stderr io.Writer, opts *FormatOptions) (int, error) {
+	if m.FormatResponse != nil {
```

Comment:
why would this be nil?

Reply:
If someone didnt configure it in their test, e.g. they didnt care about the response so just return some default instead.